handbook/handbook.sop.git-workflow.md

```diff
@@ -0,0 +1,86 @@
+
+For feature branches, we use the following conventions:
+- feat/{feature-name}
+- fix/{fix-name}
+
+### Commit Style
+- we follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) with the following tags
+- categories:
+  - feat: feature - introduce new functionality
+  - enhance: enhancement - improve existing functionality
+  - fix: make something not broken
+  - chore: backend improvements
+  - spike: not complete commit
+ 
+If you are introducing a breaking change, either add an `!` after the category or a footer at the end of your message
+
+```
+feat: allow provided config object to extend other configs
+
+BREAKING CHANGE: `extends` key in config file is now used for extending other config files
+```
+> Example from [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) licensed under [CC BY 3.0](https://creativecommons.org/licenses/by/3.0/)
+
+```
+refactor!: drop support for Node 6
+```
+> Example from [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) licensed under [CC BY 3.0](https://creativecommons.org/licenses/by/3.0/)
+
+### Pre-Requisites
+
+1. Make sure there's a [[test|dendron.dev.qa#writing-tests]] for the new feature
+1. Make sure all the existing [[tests|dendron.dev.qa#running-all-tests]] pass 
+1. Do a spot check by running your feature with our [[test Workspace|dendron.dev.qa#test-workspace]]
+1. If your change reflects documentation changes, also submit a PR to [dendron-site](https://github.com/dendronhq/dendron-site) and mention the doc PR link in your current PR
+
+
+### Creating a pull request
+
+Because we do weekly releases, we like to do a weekly cadence for pull requests as well. Pull requests for next week's release should be in 5pm local time on Thursday at the latest. 
+
+If your feature is not yet ready, you are encouraged to submit a pull request but mark it as a draft. This lets others comment on the design. If your part of the core team, this is the standard practice for weekly deliverables.
+
+If its your first pull request to Dendron, watch out for the [CLA](https://en.wikipedia.org/wiki/Contributor_License_Agreement) bot that will ask you to agree to Dendron's CLA
+
+If its your first pull request and you're on our Discord, make sure that Kevin gives you the [[horticulturalist|dendron.community.roles#horticulturalist]] role  👨‍🌾👩‍🌾
+
+After you submit your pull request, check the output of our [integration test](https://github.com/dendronhq/dendron/actions) and make sure all tests pass
+
+Optionally, ping `@Dendron Team` in the `#dev` channel of our [discord](https://discord.gg/AE3NRw9) - we usually respond to PRs within 24h
+
+
+### Merging Changes
+
+Once your feature request has been approved, it will most likely be merged by the person that is reviewing your code. 
```

Comment:
Should we add some details about the lifecycle after the merge to master?  i.e., when do we target publishing the extension; is there a time when we don't merge into master anymore (freeze on the branch)

Reply:
added some comments